tools/deploy_metrics_and_monitoring_resources.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+. $(dirname $0)/../multi-platform-support.sh
+DIR=`pwd`
+k=false
+
+usage() { echo "Usage: $0 [-c <cluster>] -n <namespace>"
+		  echo " When -c is used, Kafka cluster with metrics is deployed."
+		  1>&2; exit 1;}
+
+while getopts "n:c:" o; do
+    case "${o}" in
+        n)  NAMESPACE=${OPTARG}
+        	;;
+        c)  c=true
+            CLUSTER=${OPTARG}
+        	;; 	
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${NAMESPACE}" ] || [[ "$c" == true && -z "${CLUSTER}" ]] ; then
+    usage
+fi
+
+if [ "$c" == true ] ; then
+	# Determine whether CO is deployed in specified NS
+	lines_of_operator=`kubectl get pods -l strimzi.io/kind=cluster-operator -n $NAMESPACE | $WC -l`
+
+	if [ "$lines_of_operator" -lt "2" ]; then
+		echo "Cluster Operator is not deployed!"
+		exit 1
+	fi
+
+	echo "Deploying Kafka "$CLUSTER
+	if [[ "$OSTYPE" == "linux-gnu" ]]; then
+	    $SED "s/my-cluster/$CLUSTER/" $DIR/metrics/examples/kafka/kafka-metrics.yaml > $DIR/metrics/examples/kafka/$CLUSTER-kafka-metrics.yaml
+    elif [[ "$OSTYPE" == "darwin"* ]]; then
+        $SED '' "s/my-cluster/$CLUSTER/" $DIR/metrics/examples/kafka/kafka-metrics.yaml > $DIR/metrics/examples/kafka/$CLUSTER-kafka-metrics.yaml
+    fi
+	kubectl apply -f $DIR/metrics/examples/kafka/$CLUSTER-kafka-metrics.yaml -n $NAMESPACE
+
+	# Delay for allowing cluster operator to create the Zookeeper statefulset
+	sleep 5
+
+	zkReplicas=$(kubectl get kafka $CLUSTER -o jsonpath="{.spec.zookeeper.replicas}" -n $NAMESPACE)
+	echo "Waiting for Zookeeper cluster to be ready..."
+	readyReplicas="0"
+	while [ "$readyReplicas" != "$zkReplicas" ]
+	do
+	    sleep 2
+	    readyReplicas=$(kubectl get statefulsets $CLUSTER-zookeeper -o jsonpath="{.status.readyReplicas}" -n $NAMESPACE)
+	done
+	echo "...Zookeeper cluster ready"
+
+	# Delay for allowing cluster operator to create the Kafka statefulset
+	sleep 5
+
+	kReplicas=$(kubectl get kafka $CLUSTER -o jsonpath="{.spec.kafka.replicas}" -n $NAMESPACE)
+	echo "Waiting for Kafka cluster to be ready..."
+	readyReplicas="0"
+	while [ "$readyReplicas" != "$kReplicas" ]
+	do
+	    sleep 2
+	    readyReplicas=$(kubectl get statefulsets $CLUSTER-kafka -o jsonpath="{.status.readyReplicas}" -n $NAMESPACE)
+	done
+	echo "...Kafka cluster ready"
+
+	echo "Waiting for entity operator to be ready..."
+	kubectl rollout status deployment/$CLUSTER-entity-operator -w -n $NAMESPACE
+	echo "...entity operator ready"
+
+	rm $DIR/metrics/examples/kafka/$CLUSTER-kafka-metrics.yaml
+
+fi
+
+# Not deploying Kafka
+# Deploy Prometheus operator
+curl https://gist.githubusercontent.com/stanlyDoge/5a0b43114d9be0e72f0f4b4e3ee168a1/raw/09614357fa20efd14deffb31a06fd87aaa6dc6c8/prometheus-operator.yaml --output $DIR/metrics/prometheus-operator.yaml
```

Comment:
I do not think this should be merged into our repo. You need to handle it from some trusted source and not from a gist.